scripts/cibuild

```diff
@@ -28,6 +28,12 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         echo "Pulling down configuration settings for test environment"
         pushd "${DIR}/.."
         aws s3 cp "s3://${RF_SETTINGS_BUCKET}/.env" ".env"
+        
+        if aws s3 cp "s3://${RF_SETTINGS_BUCKET}/.sbtopts" - >> app-backend/.sbtopts; then
+            sort -u app-backend/.sbtopts | tee app-backend/.sbtopts
+        else
+            echo "Skipping appending additional .sbtopts"
```

Comment:
When do you expect to hit this condition? I have been able to hit it on a failure of `aws s3 cp`, but in that scenario I don't think the message makes sense.

Reply:
Running `cibuild` to test in my development VM will fail. 

What about something like this?

```bash
if aws s3 cp "s3://${RF_SETTINGS_BUCKET}/.sbtopts" - >> app-backend/.sbtopts; then
    echo "Merged with ${RF_SETTINGS_BUCKET} .sbtopts"
    sort -u app-backend/.sbtopts | tee app-backend/.sbtopts
fi
```

Reply:
That works. I also think it would be OK to remove the `if/fi` and just run `aws s3 cp` and `sort` back-to-back.